sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Azure.Monitor.OpenTelemetry.Exporter.Tests/TelemetryExceptionDataTests.cs

```diff
@@ -297,10 +297,16 @@ public void AggregateExceptionsWithMultipleNestedExceptionsAreTrimmedAfterReachi
 
             Assert.Equal(maxNumberOfExceptionsAllowed + 1, exceptionData.Exceptions.Count);
 
-            for (int counter = 0; counter < maxNumberOfExceptionsAllowed; counter++)
+            // The first exception is the AggregateException
+            Assert.Equal("System.AggregateException", exceptionData.Exceptions[0].TypeName);
+            Assert.Equal("0 (1) (2) (3) (4) (5) (6) (7) (8) (9) (10) (11) (12) (13) (14) (15)", exceptionData.Exceptions[0].Message);
```

Comment:
This behavior is different in net461 versus netcore. probably need conditional compile?

Reply:
Done.

Reply:
Same as https://github.com/Azure/azure-sdk-for-net/pull/33860/files#r1136312001